### ISSUE TITLE
Add `aah switch` command for switching aah between release and edge version

### ISSUE THREAD

**Issue by jeevatkm** [go-aah/aah#113]
Goal of `aah switch` command is to provide capability to switch between `aah` release and edge version.

Current approach to get aah edge version is bit complex and difficult when aah user ran into package related errors. Also updating the their codebase also pain. For example: `aah.v0` vs `aah.v0-unstable`.

So `aah switch` command will take away that pain and provides:
* Seamless way to try out aah edge version 
* I can get quick feedbacks/inputs from aah user before the actual release 😃 
* No need to update the aah application to `*-unstable`

Basically command does toggle between release and edge version
```
# posix long flag
aah switch

# posix short flag
aah s
```

To check? am I using which version...
```
# posix long flag
aah switch --whoami 

# posix short flag
aah s -w
```

_Limitations:_ 
* _This command always operates on latest version, specific version is not supported_ 
* _To begin, it works with only GOPATH. Gradually I will add vendorize support too_

**Comment by jeevatkm** [comment]
It's done, coming in v0.9 :)
